### DISCUSSION TITLE
QPKG app for QNAP Users

### DISCUSSION THREAD

**Discussion by ludoux** [openscopeproject/TrguiNG#202] [General]
### TL;DR
1. Install [TransmissionBT 4](https://www.myqnap.org/product/transmissionbt-4) made by QoolBox first.
2. **Understand that YOU TAKE YOUR OWN RISK to use the qpkg file made by a person you don't even meet or know.**
3. Install TrguiNG qpkg made by ludoux in attachments. Remember to select the correct platform. If you need another platform, reply to this post.

### Introduction
I used QDK_2.3.13 to pack the TrguiNG as a qpkg file for QNAP users to install it directly to use this webui, and it only works for TransmissionBT 4 (**not 3**).

It will rename the original `public_html` of Tr4 to `public_html_old` and create a link to TrguiNG folder, as the [Transmission4 web control](https://www.myqnap.org/product/transmission4-web-control/) do.

### Attachments
- [TrguiNG_1.3.0.b.zip](https://github.com/openscopeproject/TrguiNG/files/15345382/TrguiNG_1.3.0.b.zip)

### Version
- TrguiNG_1.3.0.b: Fixed: do not delete original webui during removal
- TrguiNG_1.3.0.a: First version based on TrguiNG_1.3.0

### Reference
https://blog.233so.com/2020/05/qdk-quick-start-guide-zh

**Comment by ludoux** [comment]
I created a [repo](https://github.com/ludoux/TrguiNG-qnap) if you are interested in it. Further release will be there.
